lib/bolt/cli.rb

```diff
@@ -131,6 +131,7 @@ def parse
                   end
 
         Bolt::Logger.configure(config.log, config.color)
+        Bolt::Logger.analytics(analytics)
```

Comment:
This needs to be updated to use assignment.

Reply:
Oh no sorry I know still working on this - I pushed too soon.